examples/conways_game_of_life_fast/model.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+from mesa import DataCollector, Model
+from mesa.space import PropertyLayer
+from scipy.signal import convolve2d
+
+
+# fmt: off
```

Comment:
If this is meant to keep the kernel matrix to looks square, wouldn't it be better to localize the `fmt: off` to that area?